test/src/main/java/org/apache/accumulo/test/ChaoticLoadBalancer.java

```diff
@@ -66,37 +67,23 @@ public void init(BalancerEnvironment balancerEnvironment) {
 
   @Override
   public void getAssignments(AssignmentParameters params) {
-    long total = params.unassignedTablets().size();
-    long avg = (long) Math.ceil(((double) total) / params.currentStatus().size());
-    Map<TabletServerId,Long> toAssign = new HashMap<>();
-    List<TabletServerId> tServerArray = new ArrayList<>();
-    for (Entry<TabletServerId,TServerStatus> e : params.currentStatus().entrySet()) {
-      long numTablets = 0;
-      for (TableStatistics ti : e.getValue().getTableMap().values()) {
-        numTablets += ti.getTabletCount();
-      }
-      if (numTablets <= avg) {
```

Comment:
When tservers had other user and metadata tablets, this would always be false for the root tablet and a tserver would never be added.